acceptance-tests/src/test/java/tech/pegasys/eth2signer/tests/UnencryptedTomlKeyLoadAndSignAcceptanceTest.java

```diff
@@ -46,9 +46,9 @@
   @ValueSource(strings = {"/signer/block", "/signer/attestation"})
   public void signDataWithKeyLoadedFromUnencryptedFile(final String artifactSigningEndpoint)
       throws Exception {
-    final String tomlKeyFilename = keyPair.publicKey().toString().substring(2);
-    final Path keyConfigFile = testDirectory.resolve(tomlKeyFilename + ".toml");
-    tomlHelpers.createUnencryptedTomlFileAt(keyConfigFile, privateKeyString);
+    final String metadataKeyFilename = keyPair.publicKey().toString().substring(2);
```

Comment:
nit: if this were configFilename, then there's no impression that it needs to be YAML (it was badly named before too)